bot/exts/evergreen/trivia_quiz.py

```diff
@@ -178,23 +396,44 @@ def check(m: discord.Message) -> bool:
                 hint_no = 0
 
                 await ctx.send(f"{msg.author.mention} got the correct answer :tada: {points} points!")
-                await self.send_answer(ctx.channel, question_dict)
+
+                await self.send_answer(ctx.channel, answers, question_dict)
                 await self.send_score(ctx.channel, self.game_player_scores[ctx.channel.id])
+
                 await asyncio.sleep(2)
 
-    @staticmethod
-    def make_start_embed(category: str) -> discord.Embed:
+    def make_start_embed(self, category: str) -> discord.Embed:
         """Generate a starting/introduction embed for the quiz."""
-        start_embed = discord.Embed(colour=discord.Colour.red())
-        start_embed.title = "Quiz game Starting!!"
-        start_embed.description = "Each game consists of 5 questions.\n"
-        start_embed.description += "**Rules :**\nNo cheating and have fun!"
-        start_embed.description += f"\n **Category** : {category}"
+        start_embed = discord.Embed(
+            colour=discord.Colour.red(),
```

Comment:
Two points here:
1. This should be a Lancebot constant not a d.py colour
2. Red usually signified failure or an error, perhaps a different colour for the start embed?

Reply:
changed to `Colours.blue`